pages/_conferenceCode/organizations/index.vue

```diff
@@ -116,14 +116,26 @@
               track-by="OrganizationTypeUID"
               placeholder="Categories"
               :options="organizationTypesOptions"
-              :multiple="false"
+              :multiple="true"
               :searchable="false"
               :clear-on-select="false"
               :close-on-select="true"
             >
+              <template slot="tag" slot-scope="{ option, remove }">
+                <span class="custom__tag">
+                  <span>{{ option.Title }}</span>
+                  <span class="custom__remove" @click="remove(option)">❌</span>
```

Comment:
don't use emoji characters no idea if it will work on the client computer

 ```suggestion
                  <span class="custom__remove" @click="remove(option)">&times;</span>
```